libraries/AP_Networking/AP_Networking.cpp

```diff
@@ -86,6 +86,20 @@ const AP_Param::GroupInfo AP_Networking::var_info[] = {
     AP_SUBGROUPINFO(param.test_ipaddr, "TEST_IP", 8,  AP_Networking, AP_Networking_IPV4),
 #endif
 
+    // @Param: OPTIONS
+    // @DisplayName: Networking options
+    // @Description: Networking options
+    // @Bitmask: 0:EnablePPP ethernet gateway
```

Comment:
Ethernet should be capitalized

Reply:
hah, I never knew it was supposed to be capitalised
https://english.stackexchange.com/questions/315198/does-ethernet-need-to-be-capitalised
changed